Special abilities v2, mostly Flak (#4)
GameBrains/er-core: modules/special-rules/pages/synapse.adoc

= Synapse

Where Synapse creatures are present, the Tyranid Hive Mind is able to impose maximum control over its swarms. 

---

Where your detachment has the {synapse} ability:

* While the detachment includes any {synapse-creature} units, apply the {dauntless} and {unbreakable} abilities.
+

> NOTE: If you lose a close combat or firefight with the detachment then it does not become Broken
> 
> {blank}[TODO: What about the Retreat move?]

Author: The revised version of the note just restates Dauntless.

This should work:

```suggestion
NOTE: If you lose a close combat or firefight with the detachment then it must still make a Retreat move, unless it also has the (no-retreat} ability.
```

Author: Moving the Retreat move note to Unbreakable and resolving this conversation.

```suggestion
```

= Synapse

Swarms of Tyranids that contain Synapse Creatures are controlled directly by the Hive Mind.

---

Apply these rules to detachments with the {synapse} ability:

> * If the detachment includes any units with the {synapse-creature} ability:
>  ** The detachment has the {dauntless} ability so automatically passes all Leadership tests.
>  ** The detachment has the {unbreakable} ability so cannot become broken.
> +
> NOTE: The detachment must retreat if it loses a close combat or firefight, but does not become broken.
> 
> * If the detachment loses all its {synapse-creature} it loses the {dauntless} and {unbreakable} abilities.

Author: Based on the Brood Telepathy conversation:

```suggestion
* While the detachment includes any {synapse-creature} units it has the {dauntless} and {unbreakable} abilities.
+
NOTE: The detachment must retreat if it loses a close combat or firefight, but does not become broken.
```

Author: Based on the updated Brood Telepathy text:

```suggestion
* While the detachment includes any {synapse-creature} units, apply the {dauntless} and {unbreakable} abilities.
+
NOTE: The detachment must retreat if it loses a close combat or firefight, but does not become broken.
```

Author: Updated suggestion added to commit batch.
